src/components/masters/program/main-content/course-cards/BaseCourseCard.jsx

```diff
@@ -82,6 +82,35 @@ class BaseCourseCard extends Component {
     return (
       <div className={classNames('card mb-4', { 'is-micromasters': !!microMastersTitle })}>
         <div className="card-body">
+          <div className="float-right mt-3 mt-xl-0">
+            <Dropdown>
+              <Dropdown.Button>
+                <FontAwesomeIcon icon={faCog} />
+              </Dropdown.Button>
+              <Dropdown.Menu>
+                <Dropdown.Item
+                  type="button"
+                  onClick={() => {
```

Comment:
We might want to extract this callback function out into a class method for a cleaner `render` method.

```js
handleEmailSettingsButtonClick = () => {
  // ...
}
```

```jsx
<Dropdown.Item
  // ...
  onClick={this.handleEmailSettingsButtonClick}
>
```